packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -9,64 +9,117 @@
         class="bg-white rounded-b max-h-72vh pb-90px overflow-scroll"
       >
         <div class="h-full">
-          <div class="p-16px">
+          <div
+            class="p-16px"
+            data-cy="release-highlights"
+          >
             <h1 class="font-medium mt-4px text-center mb-32px tracking-tighter text-22px text-gray-1000">
               {{ t('majorVersionWelcome.title') }}
             </h1>
             <div class="mb-16px">
               <ExternalLink
-                href="https://on.cypress.io/changelog#11-0-0"
+                href="https://on.cypress.io/changelog#12-0-0"
                 class="font-bold text-indigo-500"
               >
-                11.0.0
+                12.0.0
               </ExternalLink>
               <span class="font-light pl-10px text-gray-500 text-14px">
-                Released {{ versionReleaseDates['11'] }}
+                Released {{ versionReleaseDates['12'] }}
               </span>
             </div>
             <div class="children:mb-16px">
+              <p>
+                For a complete list of updates and breaking changes in v12.0.0, please review our
+                <ExternalLink href="https://on.cypress.io/changelog#12-0-0">
+                  changelog
+                </ExternalLink>.
```

Comment:
There's a space between `changelog` and the period that's caused by the linebreak before the closing tag for `ExternalLink`

![Screen Shot 2022-12-01 at 1 38 37 PM](https://user-images.githubusercontent.com/8340719/205133565-1dd5b635-d82e-4a1c-ae6e-fcc2b83699a5.png)

We have to disable a lint rule to get around it

```html
 <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
                  changelog</ExternalLink>.
```

Maybe this is a rule that we actually don't want at all and should remove.

Reply:
@marktnoonan huh, this must have auto-updated someone because it wasn't the case testing locally. How do I produce that error? I didn't have an error when running lint in this package. Is this a VS code plugin?

Reply:
fixed: https://github.com/cypress-io/cypress/pull/24925/commits/cabb14cb3cbd558f5ebb8d9910e2e3df9fc8fdca

Reply:
oh that's interesting, I don't show any diffs locally compared to what I pulled down, but I see that the extra space is not present in your video 🤔 

Messing around with window size I was able to get the period on the line below, where you had it on the same line.

![Screen Shot 2022-12-01 at 1 46 45 PM](https://user-images.githubusercontent.com/8340719/205135260-783c73f5-e481-48ef-8842-81afcb98c9c0.png)

I have no idea why it would not be the same way for you if the line break is the same in your local code, that's unexpected.

Not sure if it's a plugin, I think it's just a setting, I'll check. I get this warning if I try to change it, and then it fixes on save:

![Screen Shot 2022-12-01 at 1 52 58 PM](https://user-images.githubusercontent.com/8340719/205136342-d0b5fc2b-2bf5-4d45-b159-359e4a68d3d7.png)

Reply:
Yeah, I do think theres a plug at play.

Reply:
I guess it's both, maybe you have something locally that overrides this setting of the eslint extension:

https://github.com/cypress-io/cypress/blob/v12-splashscreen/.vscode/settings.json#L22

Still, it got fixed somewhere before you pushed the commit 🤷 

There's a similar situation at the end of the next paragraph, this doesn't happen there because the period is a part of the link, which I guess is another solution (though I like the period outside the link better).

Reply:
ewww I'll fix that.

Reply:
[`9102eee` (#24925)](https://github.com/cypress-io/cypress/pull/24925/commits/9102eee00867ae367e9e3ab93fc359b3fa67f845)